pravega/pravega
# Issue 1468: Pull retry parameters from writerConfig.

**Change log description**
Pass the EventWriterConfig retry parameters to `SegmentOutputStreamImpl`.

**Purpose of the change**
Fix #1468

**What the code does**
Actually passes the user provided parameters to where the backoff parameters are used. 

**How to verify it**
All tests should pass. Tests involving retries were manually set to turn off retries to make sure they failed (implying the setting were picked up)
